docs/csharp/tour-of-csharp/classes-and-objects.md

```diff
@@ -0,0 +1,728 @@
+---
+title: Classes and Objects in C# | A tour of the C# Language
+description: New to C#? Read this overview of classes, objects and inheritance
+keywords: .NET, csharp, class, instance, object, inheritance, polymorphism
+author: BillWagner
+manager: wpickett
+ms.date: 2016/08/10
+ms.topic: article
+ms.prod: visual-studio-dev-14
+ms.technology: devlang-csharp
+ms.devlang: csharp
+ms.assetid: 63a89bde-0f05-4bc4-b0cd-4f693854f0cd
+---
+# Classes and objects
+
+***Classes*** are the most fundamental of C#’s types. A class is a data structure that combines state (fields) and actions (methods and other function members) in a single unit. A class provides a definition for dynamically created ***instances*** of the class, also known as ***objects***. Classes support ***inheritance*** and ***polymorphism***, mechanisms whereby ***derived classes*** can extend and specialize ***base classes***.
+
+New classes are created using class declarations. A class declaration starts with a header that specifies the attributes and modifiers of the class, the name of the class, the base class (if given), and the interfaces implemented by the class. The header is followed by the class body, which consists of a list of member declarations written between the delimiters `{` and `}`.
```

Comment:
Yes. I want to defer that task until after this PR and PR #782 get merged. Then, I'll add links in existing content to match the new locations.